index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wrixby UI- An E-Commerce Store</title>
+    <title>Wrixby ECom- An E-Commerce Store</title>
```

Comment:
Tittle naming can be improved and also make it consistent for different pages.

Reply:
Sure